src/main/java/com/klarna/hiverunner/StandaloneHiveRunner.java

```diff
@@ -158,7 +159,12 @@ protected final void runTestMethod(FrameworkMethod method,
     public HiveShellContainer evaluateStatement(List<? extends Script> scripts, Object target,
         Path temporaryFolder, Statement base) throws Throwable {
         container = null;
-        FileUtil.setPermission(temporaryFolder.toFile(), FsPermission.getDirDefault());
+        File temporaryFile = temporaryFolder.toFile();
+        if (Files.notExists(temporaryFolder)) {
```

Comment:
```suggestion
        if (Files.notExists(temporaryFile)) {
```
Shouldn't we be checking the `File` not  the `Path` here?

Reply:
The method Files.notExists(Path) checks arguments of type Path, not File. Although I can change it to `if (!temporaryFile.exists()) ` if that's better